### ISSUE TITLE
Keyboard navigation doesn't work in IE

### ISSUE THREAD

**Issue by florek41** [brutaldesign/swipebox#56]
Hi !
I found that keyboard navigation doesn't work in IE (tested in IE8).
Solution (follow: http://www.quirksmode.org/dom/events/keys.html) is change bind keyup form window to document.
for example in line 360 (jquery.swipebox.js):
    $(window).bind('keyup', function(e){ ->     $(document).bind('keyup', function(e){

Then it works even in IE.

Sorry for my english ;)
